src/Scriban/Runtime/DynamicCustomFunction.cs

```diff
@@ -75,12 +75,21 @@ protected DynamicCustomFunction(MethodInfo method)
                 var lastParam = Parameters[Parameters.Length - 1];
                 if (lastParam.ParameterType.IsArray)
                 {
-                    foreach (var param in lastParam.GetCustomAttributes(typeof(ParamArrayAttribute), false))
+                    if (lastParam.ParameterType == typeof(object[]))
```

Comment:
```suggestion
                    if (lastParam.ParameterType == typeof(object[]) || lastParam.ParameterType == typeof(ScriptExpression[]))
```

Adding support for `ScriptExpression[]` as well.